tests/TestHelpers/GraphHelper.php

```diff
@@ -609,6 +609,41 @@ public static function addGroupToGroup(
 		);
 	}
 
+	/**
+	 * tries to add a group to a group through POST request
+	 *
+	 * @param string $baseUrl
+	 * @param string $xRequestId
+	 * @param string $adminUser
+	 * @param string $adminPassword
+	 * @param string $groupId
+	 * @param string $groupIdToAdd
+	 *
+	 * @return ResponseInterface
+	 * @throws GuzzleException
+	 */
+	public static function triesToaddGroupToGroupthroughPostRequest(
+		string $baseUrl,
+		string $xRequestId,
+		string $adminUser,
+		string $adminPassword,
+		string $groupId,
+		string $groupIdToAdd
+	): ResponseInterface {
+		$url = self::getFullUrl($baseUrl, 'groups/' . $groupId . '/members/$ref');
+		$payload = [
+			"@odata.id" => self::getFullUrl($baseUrl, 'groups/' . $groupIdToAdd)
+		];
+		return HttpRequestHelper::post(
+			$url,
+			$xRequestId,
+			$adminUser,
+			$adminPassword,
+			self::getRequestHeaders(),
+			\json_encode($payload)
+		);
+	}
+
```

Comment:
can you also check if we could combine this and above function into a single function by passing a method parameter. it not then its okay to have separate function.